OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ApplicationControllerTests.cs

```diff
@@ -239,12 +240,12 @@ public async Task GetByParentId_WhenParentHasNoRights_ShouldReturnBadRequest(lon
         public async Task GetByPropertyId_WhenIdIsValid_ShouldReturnOkObjectResult(long id, string property)
         {
             // Arrange
-            var filter = new ApplicationFilter{ Status = 1 };
+            var filter = new ApplicationFilter { Status = 1 };
 
             httpContext.Setup(c => c.User.IsInRole("provider")).Returns(true);
 
             providerService.Setup(s => s.GetByUserId(userId)).ReturnsAsync(provider);
-            workshopService.Setup(s => s.GetById(id)).ReturnsAsync(workshops.First());
+            //workshopService.Setup(s => s.GetById(id)).ReturnsAsync(workshops.First());
```

Comment:
(Related to this line and all subsequent similar lines)
Please don't commit commented code. 
The future of this code and intention to left it in commented state is not usually clear.

Either remove it (99% of cases) or explain commenting in prepending `//TODO` comment